tests/Integration/inc/Engine/Admin/Beacon/Beacon/insertScript.php

```diff
@@ -90,16 +93,19 @@ public function testShouldReturnBeaconScript( $config, $expected ) {
 		);
 	}
 
-	public function locale_cb() {
+	public
+	function locale_cb() {
```

Comment:
why splitting this line into two lines and check the same issue with this file methods below